klaytn/ken.yaml

```diff
@@ -0,0 +1,10 @@
+syncmode: full
```

Comment:
```suggestion
syncmode: archive
```

Reply:
I think syncmode and gcmode is different.
```
	SyncModeFlag    = TextMarshalerFlag{
		Name:   "syncmode",
		Usage:  `Blockchain sync mode ("full" or "snap")`,
		Value:  &defaultSyncMode,
		EnvVar: "KLAYTN_SYNCMODE",
	}
	GCModeFlag = cli.StringFlag{
		Name:   "gcmode",
		Usage:  `Blockchain garbage collection mode ("full", "archive")`,
		Value:  "full",
		EnvVar: "KLAYTN_GCMODE",
	}
```

Reply:
You can find gc mode flag in https://github.com/klaytn/rosetta-klaytn/pull/130/files#diff-8183fe370f9d1db240e6998e8260670d0a1b7c5e8c651eda2b3e8dae8d7e1ab2R111

Reply:
Okay got it.